iwallet/root.go

```diff
@@ -66,6 +65,7 @@ func init() {
 	rootCmd.PersistentFlags().BoolVarP(&verbose, "verbose", "v", true, "print verbose information")
 	rootCmd.PersistentFlags().BoolVarP(&elapsedTime, "elapsed_time", "", false, "print elapsed time")
 	rootCmd.PersistentFlags().StringVarP(&accountName, "account", "a", "", "which account to use")
+	rootCmd.PersistentFlags().StringVarP(&accountDir, "account_dir", "", "", "$(account_dir)/.iwallet will be used to save accounts. Default value is $HOME.")
```

Comment:
Maybe use same format as "configuration file (default $HOME/.iwallet.yaml)" like:
"$(account_dir)/.iwallet will be used to save accounts (default $HOME/.iwallet)"? No capital, no period.

Reply:
Fixed